python-lib/common.py

```diff
@@ -168,6 +168,11 @@ def assert_valid_sharepoint_path(sharepoint_path):
             )
 
 
+def assert_no_percent_in_path(path):
+    if isinstance(path, str) and "%" in path:
+        raise Exception("This plugin cannot move/rename an item if its path contains '%'")
```

Comment:
This check applies if you add a '%' too so we need the error message to explain that case too I think?
`This plugin cannot move/rename an item if its path contains '%', or to a path containing '%'"`

Reply:
Because of the UI bug I'm worried about the size of the error message on most user's environments

![Screenshot 2024-10-01 at 11 54 28](https://github.com/user-attachments/assets/d8addd9c-fff9-4cf5-bdb6-1544e1640f8c)